controllers/pod_lifecycle_manager.go

```diff
@@ -31,48 +31,48 @@ import (
 // PodLifecycleManager provides an abstraction around Pod management to allow
 // using intermediary controllers that will manage the Pod lifecycle.
 type PodLifecycleManager interface {
-	// GetInstances lists the instances in the cluster
-	GetInstances(client.Client, *fdbtypes.FoundationDBCluster, ctx.Context, ...client.ListOption) ([]FdbInstance, error)
+	// GetPods lists the instances in the cluster
+	GetPods(client.Client, *fdbtypes.FoundationDBCluster, ctx.Context, ...client.ListOption) ([]*corev1.Pod, error)
 
-	// CreateInstance creates a new instance based on a pod definition
-	CreateInstance(client.Client, ctx.Context, *corev1.Pod) error
+	// CreatePods creates a new instance based on a pod definition
+	CreatePods(client.Client, ctx.Context, *corev1.Pod) error
```

Comment:
```suggestion
	CreatePod(client.Client, ctx.Context, *corev1.Pod) error
```

We only create one Pod at a time.